TombEditor/Controls/Panel3D/ToolHandler.cs

```diff
@@ -45,35 +45,35 @@ public bool ReferenceIsOppositeDiagonalStep
                             switch (ReferenceBlock.Floor.DiagonalSplit)
                             {
                                 case DiagonalSplit.XnZp:
-                                    if (ReferencePicking.Face == BlockFace.FloorTriangle2 ||
-                                        ReferencePicking.Face == BlockFace.NegativeZ_QA ||
-                                        ReferencePicking.Face == BlockFace.NegativeZ_ED ||
-                                        ReferencePicking.Face == BlockFace.PositiveX_QA ||
-                                        ReferencePicking.Face == BlockFace.PositiveX_ED)
+                                    if (ReferencePicking.Face is BlockFace.Floor_Triangle2 or
```

Comment:
Why only some of the `||`s were converted to `or`s here?

Reply:
`or` is pattern matching and it is not a replacement for `||`. Telling whether an enum value is something or something else is pattern matching, but asking whether one condition or the other is true, needs `||`.

Reply:
Maybe we can just add some parenthesis then to make this code less confusing?